0002/README.md

```diff
@@ -0,0 +1,167 @@
+`PATCH` release for Tinkerbell can be done as required by the subprojects
+bug fixes.
+
+`MAJOR` release right now are not even evaluated because the project is too young.
+
+### GitHub release and release website
+
+Git supports tag and GitHub has the concept of a release based on a particular
+tag.
+
+The artifact generated as part of the release will be published as part of the
+release website, using as a guidance how [HashiCorp
+Release](https://releases.hashicorp.com/) works.
+
+I like the idea to use a release website vs having to push them as manifest in
+GitHub release because it gives a single place where the community can
+programmatically lookup what they are looking for, without having to look for
+repositories, versions and so on.
+
+As part of the Release page on GitHub we will publish a generated changelog.
+
+### Architectures and operating systems
+
+We know there are different architectures and Go supports cross compilations.
+When possible we should cross compile binaries at least for:
+
+* Linux
+* macOS
+* Windows
+
+For `arm` and `x86_64`.
+
+### Ownership
+
+The contributors and maintainers for each project have the responsibility for
+the quality of the release itself.
+
+Tinkerbell as a project will have its group of interest around release
+management that will take care of it (this is out of scope for this proposal but
+has to be discussed in another one).
+
+### Quality assertion
+
+Every project should be covered by tests: unit tests, functional tests and
+integration as needed.
+
+If all the test suite succeed the project can be released, otherwise tests
+should be fixed first.
+
+The documentation contains in tinkerbell.org covers the entire project and it
+has to be reviewed before any Tinkerbell release. Each Tinkerbell MINOR release will
+have its own documentation and users need a way to visualize the version they
+are looking for.
+
+### Tinkerbell
+
+Tinkerbell itself is more like a meta-project that glues all the subprojects it
+depends on. In terms of quality assertion we should have a strong and reliable
+pipeline of E2E tests to guarantee that the new version for the subprojects are
+working fine in integration.
+
+The subproject Tinkerbell depends on will be pinned as part of the Tinkerbell
+release but their version are a "suggestion". SemVer guarantees that all the
+PATCH release are working as long as the MINOR release is the same.
+
+Tinkerbell `v0.4.0` depends from boot `v1.2.0` every PATCH release for boot
+should work.
+
+SemVer guarantees that any MINOR release after the one pinned from Tinkerbell
+should work as well.
+
+Tinkerbell `v0.4.0` depends from boot `v1.2.0` every MINOR release for
+boot should work with Tinkerbell `v0.4.0` for example:
+
+* boot `v1.1.0` is not guaranteed to work
+* boot `v1.2.0` should work
+* boot `v1.2.4` should work
+* boot `v1.23.0` should work.
+
+### Support lifecycle
+
+TODO: I need some help here, personally I am thinking about 1 year (6 Tinkerbell
+releases)
```

Comment:
I think it might be a bit early to define the support lifecycle just yet at least until v1.0, maybe one of the blocking criteria for v1.0 should be that there is a defined support lifecycle.

For pre-1.0, I think it should be sufficient to say that only the current (non-pre-release) minor release is supported.